ansible/roles/common/files/path.sh

```diff
@@ -0,0 +1 @@
+export PATH=$PATH:/usr/local/flink/bin
```

Comment:
Yes, the /etc/environment file does not work for setting the $PATH, as it
gets overridden by the $PATH set in /etc/profile. I can change the path.sh
file to be a template, and get the path dynamically.

On Fri, Sep 11, 2015 at 6:04 PM, Georgios Ouzounis <notifications@github.com
> wrote:

> In ansible/roles/common/files/path.sh
> <https://github.com/grnet/okeanos-LoD/pull/81#discussion_r39280176>:
>
> > @@ -0,0 +1 @@
> > +export PATH=$PATH:/usr/local/flink/bin
>
> I think it is a bad practice to hard code the installation path
> (usr/local) of Apache Flink. Have you considered using the environment
> template file the common role already deployes?
>
>
> https://github.com/grnet/okeanos-LoD/blob/master/ansible/roles/common/templates/environment.j2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grnet/okeanos-LoD/pull/81/files#r39280176>.
>



-- 
Ioannis Tsafaras
Scientific Computing Office
Tel. +302310998988
Fax +302310999100
http://www.it.auth.gr